js/src/main/scala/com/raquo/domtypes/jsdom/defs/package.scala

```diff
@@ -37,6 +37,13 @@ package object defs {
       dom.DragEvent
     ]
 
+    type PointerEventProps[EP[_ <: dom.Event]] = generic.defs.eventProps.PointerEventProps[
+      EP,
+      dom.Event,
+      dom.PointerEvent,
+      TypedTargetPointerEvent[dom.Element],
```

Comment:
This trailing comma is failing Scala 2.11 build

Reply:
I fixed the trailing comma. But it looks like i messed up the rebase.  
I can re-create the PR if you wish.